lib/Steps/Integrations/Cordova.ts

```diff
@@ -168,47 +186,30 @@ export class Cordova extends BaseIntegration {
     });
   }
 
-  private _addNewXcodeBuildPhaseForSymbols(buildScripts: any, proj: any): void {
+  private _addNewXcodeBuildPhaseForSymbols(
+    buildScripts: any,
+    proj: any,
+    xcodeSymbolScriptPath: string,
+  ): void {
     for (const script of buildScripts) {
       if (script.shellScript.match(/SENTRY_PROPERTIES/)) {
         return;
       }
     }
-    const cwd = path.join(process.cwd(), 'sentry.properties');
+
+    const script = fs
+      .readFileSync(xcodeSymbolScriptPath, 'utf8')
+      .replace(/\\/g, '\\\\')
+      .replace(/\n/g, '\\n');
```

Comment:
![image](https://github.com/user-attachments/assets/7cd6cba2-a42a-422e-86e2-579c1cabdc15)
Diff check from the shell script file compared to the same script copy/pasted on the xcode project shows the logic above matches the xcode output.

Reply:
I my comment, I thought we would add the file to the wizard, and then copy it to the build phase.

Since we are adding the file in the Cordova SDK. https://github.com/getsentry/sentry-cordova/pull/372/files

Can we simplify the code to just executing the file https://github.com/getsentry/sentry-wizard/blob/fe849febbedb56dc803f79c79d2347b75900c032/src/react-native/xcode.ts#L189